magda-web-client/src/Components/Dataset/View/DatasetPageSuggestForm.js

```diff
@@ -85,27 +85,36 @@ export default class DatasetPageSuggestForm extends React.Component {
         };
         const formProps = {
             title: false,
+            description:
+                "Your question will be sent to the publishers of the data: " +
+                this.props.contactPoint,
```

Comment:
May need to re-phrase the text to consider the situation when `this.props.contactPoint` doesn't exist?
Otherwise, it might show as:
`Your question will be sent to the publishers of the data: undefined`

Reply:
When the primary contact point (in this case, email) doesn't exist, I've refactored it to output the remaining attributes in the modal instead. You can see this with "/dataset/ds-aurin-aurin%3Adatasource-TUA_PHIDU-UoM_AURIN_DB_sa2_incomesupport" published by GeoServer.

We have also removed the "Ask a question" button for all datasets with no contact points. This should solve for the remaining undefined contact point issues.